Dima/netflix-roulette/src/Components/Sort/Sort.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import Button from '../../Shared/Buttons/Button'
+import { parse } from 'query-string'
+import * as H from 'history'
+
+interface ISortProps {
+  handleSearchChange( sortBy: any ): void,
+  numberOfFilms: number,
+  location: H.Location
+};
+
+interface ISortState {
+  sortBy: string
+};
+
+export default class Sort extends React.Component<ISortProps, ISortState>{
+  state: ISortState = {
+    sortBy: 'relise date'
```

Comment:
You use this string several times in this file, better to move in to enum or const